### PR TITLE
Revamp PR template

### PR DESCRIPTION
**Pull request by wangannie**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Introduced a new PR template format to improve clarity and provide more prompts/guidance through comments.

### Screenshots
[//]: # "Optional but strongly recommended - Add screenshots of expected behavior - GIFs if you're feeling fancy!"

Here's what it looks like!
image

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
Would appreciate feedback on:
- clarity of wording
- if I should add details to any of the comments
- suggestions on sections to add/remove/modify

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- Use Github Actions to enforce that people have completed the Required sections

### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"
n/a

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"
The base PR template was from DC Central Kitchen: https://github.com/calblueprint/dccentralkitchen/pull/23

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🥦 CC: @sauhardjain

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
+[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"
 
-## One liner: what did you do? 🚨 
-_give a brief overview of your PR_
+## What's new in this PR
+### Description
+[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
 
+### Screenshots
+[//]: # "Optional but strongly recommended - Add screenshots of expected behavior - GIFs if you're feeling fancy!"
```
Comment (sauhardjain) [inline .github/pull_request_template.md]: Add "specifically recommended for frontend PRs"?

**Review by sauhardjain — APPROVED** [review]
Looks great - thanks for making this!

File: .github/pull_request_template.md (hunk 1 above)
Comment (wangannie) [reply .github/pull_request_template.md]: Good call -- will update.
